plugins/destination/bigquery/client/client.go

```diff
@@ -12,18 +12,21 @@ import (
 )
 
 type Client struct {
+	destination.UnimplementedUnmanagedWriter
 	destination.DefaultReverseTransformer
 	logger     zerolog.Logger
 	spec       specs.Destination
 	metrics    destination.Metrics
 	batchSize  int
 	pluginSpec Spec
+	client     *bigquery.Client
 }
 
 func New(ctx context.Context, logger zerolog.Logger, destSpec specs.Destination) (destination.Client, error) {
```

Comment:
I'm a little confused about why the linter isn't complaining about the unused `ctx` parameter - is it still running? 🤔